Makefile

```diff
@@ -80,8 +80,8 @@ FUNCTIONAL_TEST_XDC_ROOT      := ./tests/xdc
 FUNCTIONAL_TEST_NDC_ROOT      := ./tests/ndc
 DB_INTEGRATION_TEST_ROOT      := ./common/persistence/tests
 DB_TOOL_INTEGRATION_TEST_ROOT := ./tools/tests
-INTEGRATION_TEST_DIRS := $(DB_INTEGRATION_TEST_ROOT) $(DB_TOOL_INTEGRATION_TEST_ROOT)
-UNIT_TEST_DIRS := $(filter-out $(FUNCTIONAL_TEST_ROOT)% $(FUNCTIONAL_TEST_XDC_ROOT)% $(FUNCTIONAL_TEST_NDC_ROOT)% $(DB_INTEGRATION_TEST_ROOT)% $(DB_TOOL_INTEGRATION_TEST_ROOT)%,$(TEST_DIRS))
+INTEGRATION_TEST_DIRS := $(DB_INTEGRATION_TEST_ROOT) $(DB_TOOL_INTEGRATION_TEST_ROOT) ./temporaltest ./internal/temporalite
+UNIT_TEST_DIRS := $(filter-out $(FUNCTIONAL_TEST_ROOT)% $(FUNCTIONAL_TEST_XDC_ROOT)% $(FUNCTIONAL_TEST_NDC_ROOT)% $(INTEGRATION_TEST_DIRS)%,$(TEST_DIRS))
```

Comment:
Please double check me on this diff. I needed to update `UNIT_TEST_DIRS` to filter out all of `INTEGRATION_TEST_DIRS`.

Reply:
I think this change might be wrong: previously it had DB_INTEGRATION_TEST_ROOT and DB_TOOL_INTEGRATION_TEST_ROOT both followed by a `%`, which in gnu make pattern syntax is a wildcard that matches anything, i.e. anything below those dirs. Now it uses INTEGRATION_TEST_DIRS, which expands to four things, plus a `%`, but I suppose that means only the last of them gets the `%`, so unit tests will include things they shouldn't.

There's probably some fancy gnu make syntax to make it do what you want, but in the short term I think just repeating the four items each followed by `%` is fine